geps/gep-1897/index.md

```diff
@@ -150,10 +154,10 @@ Because naming is hard, a new name may be
 substituted without blocking acceptance of the content of the API change.
 
 The selection of the applicable Gateway API persona is important in the design of BackendTLSPolicy, because it provides
-a way to explicitly describe the _expectations_ of the connection to the application.  BackendTLSPolicy is configured
-by the application developer Gateway API persona to signal what the application developer _expects_ in connections to
-the application, from a TLS perspective.  Only the application developer can know what the application expects, so it is
-important that this configuration be managed by that persona.
```

Comment:
Why can't this remain?  It was to clarify how and why the feature should be used.